src/index.js

```diff
@@ -31,6 +36,9 @@ Dotdotdot.prototype.dotdotdot = function(container) {
     });
   }
 };
+Dotdotdot.prototype.update = function() {
```

Comment:
Is this helper method needed? Maybe line 12 could be just `this.update = this.forceUpdate.bind(this);`